File: B/timBola_Java/Team.java
public class Team {

	// class attribute
	private String teamName;
	private String country;
	private String since;
	private String[] player;

	private Stadium st;

	// empty consturctors
	Team() {

	}

	// constructors
	Team(String teamName, String country, String since, String[] player) {
		this.teamName = teamName;
		this.country = country;
		this.since = since;
		this.player = player;
	}

	// setter and getter for all the attribute
	public void setTeamName(String teamName) {
		this.teamName = teamName;
	}

	public String getTeamName() {
		return teamName;
	}

	public void setCountry(String country) {
		this.country = country;
	}

	public String getCountry() {
		return country;
	}

	public void setSince(String since) {
		this.since = since;
	}

	public String getSince() {
		return since;
	}

	public void setSt(String stadiumName, int capacity, String stadiumLocation) {
		st = new Stadium();
		st.setStadiumLocation(stadiumLocation);
		st.setcapacity(capacity);
		st.setStadiumName(stadiumName);
	}

	public Stadium getSt() {
		return st;
	}

	public void setPlayer(String[] player) {
		this.player = player;
	}

	public String[] getPlayer() {
		return player;
	}

	class Stadium {

		// stadiumclass attribute
		public String stadiumName;
		public int capacity;
		public String stadiumLocation;

		// constructors
		Stadium() {

		}

		Stadium(String stadiumName, int capacity, String stadiumLocation) {
			this.stadiumLocation = stadiumLocation;
			this.capacity = capacity;
			this.stadiumName = stadiumName;
		}

		// setter and getter
		public void setStadiumLocation(String stadiumLocation) {
			this.stadiumLocation = stadiumLocation;
		}

		public String getStadiumLocation() {
			return stadiumLocation;
		}

		public void setStadiumName(String stadiumName) {
			this.stadiumName = stadiumName;
		}

		public String getStadiumName() {
			return stadiumName;
		}

		public void setcapacity(int capacity) {
			this.capacity = capacity;
		}

		public int getcapacity() {
			return capacity;
		}

		// method for show the stadium class result
		public void ShowStadium() {
			System.out.println("      --------");
			System.out.println("      HOMEBASE");
			System.out.println("      --------");
			System.out.println("Stadium Name: " + this.getStadiumName());
			System.out.println("Capacity: " + this.getcapacity());
			System.out.println("Location: " + this.getStadiumLocation());
		}

	}

	// method for showing the result
	public void ShowTeam() {
		System.out.println("Team Name: " + this.getTeamName());
		System.out.println("Country: " + this.getCountry());
		System.out.println("Established: " + this.getSince());
		System.out.println("Player:");
		for (String ply : this.getPlayer()) {
			System.out.println("    - " + ply);
		}
	}

}

File: B/timBola_Python/Team.py
from mimetypes import init


class Team:
    # private attribute class
    __teamName = ""
    __country = ""
    __since = ""
    __player = ""

    def __init__(self, teamName="", country="", since="", player=""):
        self.__teamName = teamName
        self.__country = country
        self.__since = since
        self.__player = player

    def setTeamName(self, teamName):
        self.__teamName = teamName

    def getTeamName(self):
        return self.__teamName

    def setCountry(self, country):
        self.__country = country

    def getCountry(self):
        return self.__country

    def setSince(self, since):
        self.__since = since

    def getSince(self):
        return self.__since

    def setPlayer(self, player):
        self.__player = player

    def getPlayer(self):
        return self.__player

    class Stadium:
        __stadiumName = ""
        __capacity = 0
        __stadiumLocation = ""

        def __init__(self, stadiumName="", capacity=0, stadiumLocation=""):
            self.__stadiumName = stadiumName
            self.__capacity = capacity
            self.__stadiumLocation = stadiumLocation

        def setStadiumName(self, stadiumName):
            self.__stadiumName = stadiumName

        def getStadiumName(self):
            return self.__stadiumName

        def setCapacity(self, capacity):
            self.__capacity = capacity

        def getCapacity(self):
            return self.__capacity

        def setStadiumLocation(self, stadiumLocation):
            self.__stadiumLocation = stadiumLocation

        def getStadiumLocation(self):
            return self.__stadiumLocation

        def ShowStadium(self):
            print("       --------")
            print("       HOMEBASE")
            print("       --------")
            print("Stadium Name: " + self.__stadiumName)
            print("Capacity: " + str(self.__capacity))
            print("Stadium Location: " + self.__stadiumLocation)

    __st = Stadium()

    def setStadium(self, stadiumName, capacity, stadiumLocation):
        self.__st.setStadiumName(stadiumName)
        self.__st.setStadiumLocation(stadiumLocation)
        self.__st.setCapacity(capacity)

    def getStadium(self):
        return self.__st

        # print method

    def ShowTeam(self):
        print("Team Name: " + self.getTeamName())
        print("Country: " + self.getCountry())
        print("Established: " + self.getSince())
        print("Player:")
        for i in self.getPlayer():
            print("    - " + i)
